README.md

```diff
@@ -1,107 +1,34 @@
 [![Project](https://img.shields.io/badge/Project-Holochain-blue.svg?style=flat-square)](http://holochain.org/)
 [![Discord](https://img.shields.io/badge/Discord-blue.svg?style=flat-square)](https://discord.gg/k55DS5dmPH)
 [![License: CAL 1.0](https://img.shields.io/badge/License-CAL%201.0-blue.svg)](https://github.com/holochain/cryptographic-autonomy-license)
-![Test](https://github.com/holochain/holochain-client-js/actions/workflows/test.yml/badge.svg?branch=main)
+![Tests](https://github.com/holochain/holochain/actions/workflows/release.yml/badge.svg)
 [![Twitter Follow](https://img.shields.io/twitter/follow/holochain.svg?style=social&label=Follow)](https://twitter.com/holochain)
 
+
 # Holochain
 
 Holochain is an open-source framework to develop peer-to-peer applications with high levels of security, reliability, and performance.
 
-This repository contains the Holochain core libraries.
-
-## Running Holochain applications (hApps)
-
-[Holochain Launcher](https://github.com/holochain/launcher), the distribution of Holochain for end-users, is supported on all three major desktop platforms.
-Holochain apps can be installed and run with it. Refer to the Github repository for instructions on how to obtain and use it.
-
-## Holochain Development
-
 ### Code Status
 
-This code is in Alpha. It is not for production use, and currently makes no security guarantees. We will be frequently and heavily restructuring code APIs and data chains until Beta.
-
-### Supported OSes
+This code is in initial Beta release. It's ready for limited production use. It is still undergoing security reviews and hardening of network protocols. 
 
-#### Linux
+## Running Holochain Applications (hApps)
 
-* Launcher binaries
-* binaries via crates.io
-* nix package and holonix shell, including execution of binary within holonix CI
-* full functionality test on CI
+This repository contains the Holochain core libraries, not the runtime intended for end-users of Holochain applications.
 
-#### macOS (x86-64 only, no ARM)
+If you just want to install and run Holochain and apps from its AppStore, download the latest release of [Holochain Launcher](https://github.com/holochain/launcher/releases) appropriate for Linux, Mac, or Windows. 
```

Comment:
Are we calling it the "AppStore"? Otherwise it could be just "app store".

Reply:
There is the intention to do that at some point.